packages/mds-geography-author/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@mds-core/mds-geography-author",
+  "description": "MDS Geography Author API",
+  "version": "0.0.1",
+  "private": true,
+  "licence": "Apache-2.0",
+  "engineStrict": true,
+  "engines": {
+    "node": ">= 8.0.0",
```

Comment:
engines in here seems unnecessary (we don't have it in anything but policy/geo stuff FWIW) and we also definitely wouldn't want it to be a non-absolute value.